catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-forms.less

```diff
@@ -0,0 +1,36 @@
+@{customElementNamespace}search-form {
+
+    display: inline-block;
+    padding: @minimumSpacing;
+    margin: @minimumSpacing;
+    width: 300px;
```

Comment:
Can we update these to use a multiple of minimumButtonSize?  Then it will respond to the font size settings and the spacing settings.

Reply:
Done